packages/react/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "baseUrl": ".",
     "declaration": true,
     "esModuleInterop": true,
-    "jsx": "react-jsx",
```

Comment:
For my knowledge, can you briefly explain why this had to be changed?

Reply:
This is controls how JSX constructs are emitted in JavaScript files. It has better details here <- https://www.typescriptlang.org/tsconfig#jsx

Reply:
TIL!